manifests/server/read_database_ini.pp

```diff
@@ -42,12 +45,16 @@
       mode   => '0600';
     }
 
+    $ini_setting_require = str2bool($database_validate) ? {
+      false => undef,
+      default  => Class['puppetdb::server::validate_read_db'],
+    }
     # Set the defaults
     Ini_setting {
       path    => "${confdir}/read_database.ini",
       ensure  => present,
       section => 'read-database',
-      require => Class['puppetdb::server::validate_db'],
```

Comment:
we're replacing this with puppetdb::server::validate_read_db -- was that an existing bug or is this changing behavior? ping @kbarber